twisted/mail/alias.py

```diff
@@ -742,7 +742,8 @@ def __init__(self, items, *args):
                 except:
                     log.err("Invalid filename in alias file %r" % (addr[1:],))
                 else:
-                    addr = ' '.join([l.strip() for l in f])
+                    with f:
```

Comment:
well... we need it to not catch join errors ... and strip() errors... so is ok to have it this way. sorry for the noise